### DISCUSSION TITLE
Add the emoticons / emojis picker to the tiny mce editor

### DISCUSSION THREAD

**Discussion by Guisardo** [directus/directus#14294] [Feature Requests]
As a site editor I would like to have the posibility to use an emoticons/emoji selector within the rich text editor of the collection items to be able to write more friendly content.

The tiny mce already have an emoji picker:
https://www.tiny.cloud/blog/emoji-picker/

But there seems to be no way to enable it with the current Directus configurations.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
